plugin/src/main/kotlin/com/ritense/plugin/domain/ActivityType.kt

```diff
@@ -16,6 +16,22 @@
 
 package com.ritense.plugin.domain
 
+import mu.KotlinLogging
+
 enum class ActivityType {
-    SERVICE_TASK
-}
+    SERVICE_TASK;
+
+    companion object {
```

Comment:
if we're doing this i'd rather have the corresponding key be part of the enum, instead of having another method having that knowledge. so something like this:

```
enum class ActivityType(
    val bpmnModelValue: String
) {

SERVICE_TASK("bpmn:ServiceTask")
```

Reply:
I'll have a look